cfme-performance/conf/cfme_performance.yml

```diff
@@ -95,6 +95,25 @@ workloads:
         refresh_sleep_time: 600
         total_time: 14400
         grafana_dashboard: cfme-vmware-redhat-system-performance
+  test_provisioning:
```

Comment:
Lets move this example down to the bottom of the file.  I would like to keep the idea that the hardest to run scenarios are at the bottom of the file thus I believe the current line up should be:

1. idle workloads
2. refresh workloads
3. cap & util workloads
4. SSA workload(s)
5. provisioning workloads